packages/node_modules/@webex/internal-plugin-wdm/src/device/device.js

```diff
@@ -255,11 +263,32 @@ const Device = WebexPlugin.extend({
       });
     });
 
+    this.isInNetwork = false;
     this.on('change:serviceHostMap', this._updateServiceCatalog);
 
-    this.listenToAndRun(this, 'change:intranetInactivityCheckUrl', () => this._resetLogoutTimer());
-    this.listenToAndRun(this, 'change:intranetInactivityDuration', () => this._resetLogoutTimer());
-    this.listenTo(this.webex, 'user-activity', () => { this.lastUserActivityDate = Date.now(); });
+    this.listenToAndRun(this, 'change:intranetInactivityCheckUrl', () => { this._resetLogoutTimer(); this._inNetworkResetLogoutTimer(); });
+    this.listenToAndRun(this, 'change:intranetInactivityDuration', () => { this._resetLogoutTimer(); this._inNetworkResetLogoutTimer(); });
+    this.listenToAndRun(this, 'change:inNetworkInactivityDuration', () => { this._resetLogoutTimer(); this._inNetworkResetLogoutTimer(); });
+    this.listenTo(this.webex, 'user-activity', () => {
+      this.lastUserActivityDate = Date.now();
+      this.lastInNetworkUserActivityDate = Date.now();
+    });
+    /**
+     * Meeting started and ended event listenrs
+     * When client enters into the meeting set isInMeeting to true
+     * This tells also triggers both the timers inturn un-setting the timers until meeting ends
+     * On meeting end/clean-up the timers are reset.
+     */
+    this.listenTo(this.webex, 'meeting-started', () => {
```

Comment:
Spaces instead of dashes are good for custom events, e.g., `'meeting started'`, [more examples](https://github.com/webex/webex-js-sdk/wiki/plugin-meetings-events)